docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -34,6 +34,9 @@ cluster sizing video]. As you test different shard configurations, use {kib}'s
 {kibana-ref}/elasticsearch-metrics.html[{es} monitoring tools] to track your
 cluster's stability and performance.
 
+The performance of an {es} node is often limited by the performance of the underlying storage. 
```

Comment:
nit Maybe this paragraph can be refactored with the same paragraph of datatiers.asciidoc and node.asciidoc?